configs/webpack.config.renderer.dev.babel.js

```diff
@@ -10,7 +10,7 @@ import fs from 'fs';
 import webpack from 'webpack';
 import chalk from 'chalk';
 import merge from 'webpack-merge';
-import { spawn, execSync } from 'child_process';
+import { spawn, execSync } from 'cross-spawn';
```

Comment:
Why this change? Not sure if we’re using this dep

Reply:
It was already in the `package.json` and I thought it might have been needed. 

Is it not the case?

Reply:
Might have been accidentally added. It’s not used anywhere

Reply:
Actually we don't need `cross-spawn` for executing process, `child_process` is enough